eoscostarica/guias.eoscostarica.io
# docs(frontend): add 'preguntas-frecuentes' section to spanish 'guias'…

### What does this PR do?
Resolves #161 

### Screenshot
![image](https://user-images.githubusercontent.com/5632966/103849529-8d299480-506a-11eb-98a4-b6f938b675b1.png)


### Steps to test
1. Go to Preguntas frecuentes inside of Herramientas tag
1. Check content is right

#### CheckList
- [ ] Seccion preguntas frequentes bajo seccion herramientas
- [ ] Contenido de muestra
